### PR TITLE
Export sensi results as csv

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

File: src/components/results/sensitivity-analysis/sensitivity-analysis-result.js [hunk 1]
```diff
@@ -46,6 +53,10 @@ const SensitivityAnalysisResult = ({
     sortProps,
     filterProps,
     isLoading,
+    handleCsvHeadersChange,
+    exportResultAsCsv,
```
Comment (thangqp) [inline src/components/results/sensitivity-analysis/sensitivity-analysis-result.js:57]: In my opinion, it is more conventional with

onCsvHeadersChange: handleCsvHeadersChange, 
onExportResultAsCsv: handleExportResultAsCsv,

And add 4 new props to propTypes

File: src/components/results/sensitivity-analysis/sensitivity-analysis-result.js [hunk 2]
```diff
@@ -186,11 +197,25 @@ const SensitivityAnalysisResult = ({
         suppressMultiSort: true,
     };
 
-    const onGridReady = useCallback((params) => {
-        if (params.api) {
-            params.api.sizeColumnsToFit();
-        }
-    }, []);
+    const onGridReady = useCallback(
```
Comment (thangqp) [inline src/components/results/sensitivity-analysis/sensitivity-analysis-result.js:200]: idem, handleGridReady

File: src/components/results/sensitivity-analysis/sensitivity-analysis-result.js [hunk 3]
```diff
@@ -186,11 +197,25 @@ const SensitivityAnalysisResult = ({
...
+        (params) => {
+            if (params.api) {
+                params.api.sizeColumnsToFit();
+                handleCsvHeadersChange(getDisplayedColumns(params));
+            }
+        },
+        [handleCsvHeadersChange]
+    );
+
+    const onGridColumnsChanged = useCallback(
```
Comment (thangqp) [inline src/components/results/sensitivity-analysis/sensitivity-analysis-result.js:210]: idem, handleGridColumnsChanged

**Review by thangqp — APPROVED** [review]
Code OK
Test OK

**Review by achour94 — APPROVED** [review]

**Review by achour94 — APPROVED** [review]

File: src/components/results/sensitivity-analysis/sensitivity-analysis-result-tab.js [hunk 4]
```diff
@@ -90,14 +158,27 @@ const SensitivityAnalysisResultTab = ({ studyUuid, nodeUuid }) => {
             />
             {sensiResultKind.includes(sensiKind) && (
                 <>
-                    <Tabs
-                        value={nOrNkIndex}
-                        onChange={handleSensiNOrNkIndexChange}
+                    <div
```
Comment (thangqp) [inline src/components/results/sensitivity-analysis/sensitivity-analysis-result-tab.js:161]: We no longer use div ???

**Pull request merged**